diff --git a/node_modules/react-slick/lib/inner-slider.js b/node_modules/react-slick/lib/inner-slider.js
index 2535e6b..f5136bb 100644
--- a/node_modules/react-slick/lib/inner-slider.js
+++ b/node_modules/react-slick/lib/inner-slider.js
@@ -408,7 +408,7 @@ var InnerSlider = exports.InnerSlider = function (_React$Component) {
       });
       onLazyLoad && slidesToLoad.length > 0 && onLazyLoad(slidesToLoad);
       _this.setState(state, function () {
-        asNavFor && asNavFor.innerSlider.state.currentSlide !== currentSlide && asNavFor.innerSlider.slideHandler(index);
+        asNavFor.current && asNavFor.current.innerSlider.state.currentSlide !== currentSlide && asNavFor.current.innerSlider.slideHandler(index);
         if (!nextState) return;
         _this.animationEndCallback = setTimeout(function () {
           var animating = nextState.animating,
